app/controllers/publishers/vacancies/activity_log_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Publishers::Vacancies::ActivityLogController < Publishers::Vacancies::BaseController
+  helper_method :activity_log
+
+  def show; end
+
+  private
+
+  def activity_log
+    vacancy.versions.map { |version| { created_at: version.created_at, publisher: version.actor, changeset: version.changeset } }.reverse
```

Comment:
Still think this isn't necessary 🤔 We should probably just have something like this in the show:

```ruby
def show
  @versions = vacancy.versions.order(created_at: :desc)
end
```

And then iterate over `@versions` in the view.